### PR TITLE
Add new runbook for Register a new `gov.uk` subdomain

### PR DESCRIPTION
**Pull request by AntonyBishop**
## 👀 Purpose

- This PR adds a new runbook for Registering a new `gov.uk` subdomain, and removes out of date "DNS cutover" runbook.

## ♻️ What's changed

- Add Register New `gov.uk` subdomain runbook
- Add Register New `gov.uk` subdomain runbook to index
- Remove Support DNS Cutover Runbook
- Remove Support DNS Cutover Runbook from index

### CODE REVIEW

**Review by andyrogers1973 — APPROVED** [review]
lgtm

File: source/documentation/dns/register-new-gov-uk-subdomain.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+owner_slack: "#operations-engineering-alerts"
+title: Register new gov.uk subdomain
+last_reviewed_on: 2024-09-018
```
Comment (jasonBirchall) [inline source/documentation/dns/register-new-gov-uk-subdomain.html.md.erb:4]: ouchies

Comment (jasonBirchall) [reply source/documentation/dns/register-new-gov-uk-subdomain.html.md.erb:4]:
```suggestion
last_reviewed_on: 2024-09-18
```

Comment (AntonyBishop) [reply source/documentation/dns/register-new-gov-uk-subdomain.html.md.erb:4]: Thank you!

**Review by andyrogers1973 — APPROVED** [review]
looks good

**Pull request merged**